### PR TITLE
My account page bugs

### PR DESCRIPTION
**Pull request by daisyzarzo**
relates to https://github.com/bespoken/dashboard/issues/803
and https://github.com/bespoken/dashboard/issues/802

### CODE REVIEW

File: src/components/UserControl.tsx [hunk 1]
```diff
@@ -24,8 +24,9 @@ export default class UserControl extends React.Component<UserProps, any> {
     button.click();
   }
 
-  handleGoToClick = (url: string) => {
+  handleGoToClick = async (url: string) => {
     window.open(url, "_blank");
+    await setTimeout(() => { /*Your Code*/ }, 200);
```
Comment (jperata) [inline src/components/UserControl.tsx]: this is not going to wait before doing the next part, 
await only work with promises, and setTimeout works by executing the internal part , if you need to wait,
     setTimeout(() => { this.emulateMenuClick()}, 200);

**Comment by jperata** [comment]
paste screenshots of the corrected changes

**Comment by daisyzarzo** [comment]
image

**Comment by daisyzarzo** [comment]
FIREFOX
image

**Comment by daisyzarzo** [comment]
JUST ONE TAB
image

File: src/components/UserControl.tsx [hunk 2]
```diff
@@ -24,9 +24,9 @@ export default class UserControl extends React.Component<UserProps, any> {
     button.click();
   }
 
-  handleGoToClick = (url: string) => {
+  handleGoToClick = async (url: string) => {
     window.open(url, "_blank");
-    this.emulateMenuClick();
+    await setTimeout(() => { this.emulateMenuClick(); }, 200);
```
Comment (jperata) [inline src/components/UserControl.tsx]: remove async and await, this is not a promise

**Pull request merged**